representatives.py

```diff
@@ -4,18 +4,20 @@
 import requests
 import json
 import argparse
+from typing import List, Dict
 
 import peercrawler
 import pynanocoin
 import acctools
 import constants
 
-RPC_URL = 'http://[::1]:7076'
-#RPC_URL = 'https://mynano.ninja/api/node'
+
+RPC_URL = "https://mynano.ninja/api/node"
```

Comment:
Why are you changing single quotes to double quotes?
I generally prefer single quotes. I find them neater.

Reply:
As you're probably aware, several languages use singles quotes for declaring a char and doubles quotes for strings, while in Python there's no such different, so I'm just used to double quotes strings from working on C# projects.